ttml.go

```diff
@@ -380,7 +413,8 @@ func ReadFromTTML(i io.Reader) (o *Subtitles, err error) {
 
 		// Unmarshal items
 		var items = TTMLInItems{}
-		if err = xml.Unmarshal([]byte("<span>"+ts.Items+"</span>"), &items); err != nil {
+		decoder := xml.NewTokenDecoder(newHandleBrTokenReader(strings.NewReader("<p>" + ts.Items + "</p>")))
```

Comment:
Could you simplify to `if err = newTTMLXmlDecoder(ts).Decode(&items); err != nil { ... }`?